### DISCUSSION TITLE
Wiki-links with directory path?

### DISCUSSION THREAD

**Discussion by igtsimoes** [notaapp/talk#391] [Questions] [answered]
Hello. 

I have some notes that shows wiki-links with the directories to the notes, like this: 

<img width="365" alt="image" src="https://user-images.githubusercontent.com/64813530/231024703-c372d1cb-6c62-4d24-9095-dcb7cfd5d24e.png">

But a few days ago, the links were just the note name, like this:

<img width="139" alt="image" src="https://user-images.githubusercontent.com/64813530/231024855-ad67bef5-0c98-4ca2-8eed-68ed58c106bc.png">

Showing only the note name is much cleaner... how do I do to always have this behavior? 

Thank you!

**Comment by astoilkov** [comment, marked as answer]
Hmm, that's probably because the wiki link itself is written with it's full path `[[path/to/folder/file.md]]`. Is it possible you have disabled "Short Paths in Wiki-Links" option?

<img width="475" alt="CleanShot 2023-04-13 at 14 52 41@2x" src="https://user-images.githubusercontent.com/884810/231750390-93b8efac-ef5e-48ed-84a7-f889fb1ab56d.png">

**Reply by igtsimoes** [reply]
Thank you @astoilkov ! That was it... playing around with the option and didn't realise I had to turn that back on.
